data/reusables/sponsors/tax-form-information-dev.md

```diff
@@ -1,4 +1,4 @@
 For more information about the tax forms you need to complete, see the following instructions on the United States Internal Revenue Service website.
 
-* [Instructions for Form W-9](https://www.irs.gov/pub/irs-pdf/iw9.pdf), for US residents
-* [Instructions for Form W-8BEN](https://www.irs.gov/pub/irs-pdf/iw8ben.pdf), for non-US residents
+* [Instructions for Form W-9](https://www.irs.gov/pub/irs-pdf/iw9.pdf), for US residents and non-resident US citizens
+* [Instructions for Form W-8BEN](https://www.irs.gov/pub/irs-pdf/iw8ben.pdf), for non-US residents and residents
```

Comment:
I am not sure we need to merge this one.

Reply:
@catsintents @slincoln75 Thank you both for these comments! 

@gagan-bhullar-tech Would you be able to update the PR per the comments above? Once that's complete, we merge this PR 💛

Reply:
@nguyenalex836 its updated.